api/api/views/viewsets/mailing_list.py

```diff
@@ -15,6 +17,32 @@ def create(self, request, *args, **kwargs):
         serializer.is_valid(raise_exception=True)
         self.perform_create(serializer)
         headers = self.get_success_headers(serializer.data)
+        email_address = serializer.validated_data['email']
+
+        # Create SendgridClient and send welcome email
+        try:
+            email_client = SendgridClient(recipient=email_address)
+            email_client.mail.template_id = SendgridParameters.TEMPLATE_ID_WELCOME
+            email_client.set_template_data({
```

Comment:
we have a template for this already, you don't have to add all this text this is not necessary